app/controllers/delivery_zones_controller.rb

```diff
@@ -0,0 +1,29 @@
+class DeliveryZonesController < ApplicationController
+  def index
+    @delivery_zones = DeliveryZone.all
+  end
+
+  def show
+    @delivery_zone = DeliveryZone.find_by!(zipcode: params[:id])
+  end
+
+  def create
+    @delivery_zone = DeliveryZone.new(delivery_zone_params)
+
+    if @delivery_zone.save
+      redirect_to delivery_zone_url(@delivery_zone)
```

Comment:
@kenyonj according to the docs for [`redirect_to`](http://api.rubyonrails.org/classes/ActionController/Redirecting.html#method-i-redirect_to):

> Record - The URL will be generated by calling url_for with the options, which will reference a named URL for that record.

Looks like the answer is yes.